gocourse17/cmd/client/main.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/vicuani/go_course/gocourse17/internal/grpc/grpcapi"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+type TaxiServiceClient struct {
+	client grpcapi.TaxiServiceClient
+}
+
+func (t *TaxiServiceClient) sendCargoStateRating(driverID int32, cargoState grpcapi.Rating) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+	defer cancel()
+
+	req := &grpcapi.EvaluateCargoStateRequest{
+		DriverId:   driverID,
+		CargoState: cargoState,
+	}
+
+	res, err := t.client.EvaluateCargoState(ctx, req)
+	if err != nil {
+		cancel()
+		log.Fatalf("could not submit cargo state rating: %v", err)
+	}
+
+	log.Printf("Response from server: %s\n", res.Message)
+}
+
+func (t *TaxiServiceClient) sendDriverServiceRating(driverID int32, driverService grpcapi.Rating) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+	defer cancel()
+
+	req := &grpcapi.EvaluateDriverServiceRequest{
+		DriverId:      driverID,
+		DriverService: driverService,
+	}
+
+	res, err := t.client.EvaluateDriverService(ctx, req)
+	if err != nil {
+		cancel()
+		log.Fatalf("could not submit driver service rating: %v", err)
+	}
+
+	log.Printf("Response from server: %s\n", res.Message)
+}
+
+func (t *TaxiServiceClient) sendDeliverySpeedRating(driverID int32, deliverySpeed grpcapi.Rating) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+	defer cancel()
+
+	req := &grpcapi.EvaluateDeliverySpeedRequest{
+		DriverId:      driverID,
+		DeliverySpeed: deliverySpeed,
+	}
+
+	res, err := t.client.EvaluateDeliverySpeed(ctx, req)
+	if err != nil {
+		cancel()
+		log.Fatalf("could not submit delivery speed rating: %v", err)
+	}
+
+	log.Printf("Response from server: %s\n", res.Message)
+}
+
+func (t *TaxiServiceClient) getDriverReviews(driverID int32) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+	defer cancel()
+
+	req := &grpcapi.DriverReviewsHistoryRequest{
+		DriverId: driverID,
+	}
+
+	res, err := t.client.DriverReviewsHistory(ctx, req)
+	if err != nil {
+		cancel()
+		log.Fatalf("could not get driver reviews: %v", err)
```

Comment:
Думаю, можна замінити `Fatalf` на `Panicf` і прибрати всі `cancel()`:
```suggestion
		log.Panicf("could not get driver reviews: %v", err)
```